src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1119,6 +1119,34 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
             }
         }
 
+        [OuterLoop] // takes several seconds
+        [Theory]
+        [InlineData(302, false)]
+        [InlineData(307, true)]
+        public async Task PostAsync_Redirect_LargePayload(int statusCode, bool expectRedirectToPost)
+        {
+            using (var fs = new FileStream(Path.Combine(Path.GetTempPath(), Path.GetTempFileName()), FileMode.Create, FileAccess.ReadWrite, FileShare.None, 0x1000, FileOptions.DeleteOnClose))
+            {
+                string contentString = string.Join("", Enumerable.Repeat("Content", 100000));
```

Comment:
700k characters?  Do computers even have that much RAM? :smile:

I'm a little surprised that you didn't just call fs.Write 100000 times with the UTF-32 string "Content".

Not really important for a test, I suppose.